.github/workflows/ci.yml

```diff
@@ -13,6 +13,11 @@ on:
         description: "Docker hub image name taken from https://hub.docker.com/r/statusteam/nim-waku/tags. Format: statusteam/nim-waku:v0.19.0"
         required: false
         type: string
+  workflow_call:
```

Comment:
Is is accesible now from `nwaku`? I am not sure the organizational setting allows it now

Reply:
@vpavlin  is doing the changes on nwaku side and we'll check if we can use reusable workflows across repos. Worst case we need to use github api

Reply:
> even outside of `waku-org`

huh, we need to fix it, I guess the setting is too generous

@fryorcraken can you limit execution only to the [same org](https://docs.github.com/en/actions/using-workflows/reusing-workflows#access-to-reusable-workflows)?

Reply:
Why? It's a public repo, which is IMO how this gets "pulled in" to your workflow execution - the actual execution happens on caller runners, so it does not really influence you/the js-waku repo

Reply:
if the context is in another repo - then sounds good, I am worried about triggering `release` step. shouldn't be the case but don't know exactly

Reply:
The release step requires the CI_TOKEN secret which my (or anyone else's repo) does not know, so unless you tell me that secret, I cannot actually successfully execute that step:) (which is why we made it conditional)

Reply:
seems to be the case too, but because we already talk about this and that - it might be better to try to make a re-usable aciton

@fbarbu15 maybe you looked into that?

Reply:
I agree that splitting this WF up into smaller chunks would make sense, but would it be ok to merge it as is and the iterate over it? I'd like to get this in on our side before our release next week (i.e. ideally merge both PRs today)

Reply:
Considering that we need to run just `node and node-optional tests`, and exclude any other jobs, then indeed using reusable workflows for those steps would be the best approach. 
In that case the `nwaku` CI will call the `node.yml` and `node-optional.yml` instead of calling the whole `ci.yml`
Would that work for you @vpavlin ?

Reply:
Yeah, that would be fine for me

Reply:
I'm ok to merge it and create a new PR with the splitting. @weboko  what about you?

Reply:
oh, I thought you could merge yourself
I agree with follow-up to this later, let's track it here https://github.com/waku-org/js-waku/issues/1548